pylint/checkers/imports.py

```diff
@@ -575,7 +576,7 @@ def _check_imports_order(self, _module_node):
         std_imports = []
         third_party_imports = []
         first_party_imports = []
-        # need of a list that holds third or first party ordered import
+        # need of a list that holds third or first party ordered import
```

Comment:
Why these space changes for?

Reply:
Pep8: https://www.python.org/dev/peps/pep-0008/#inline-comments